src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/DiaryOverviewTableViewController.swift

```diff
@@ -74,9 +77,9 @@ class DiaryOverviewTableViewController: UITableViewController {
 	override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
 		switch indexPath.section {
 		case 0:
-			return descriptionCell(at: indexPath)
+			return descriptionCell(forRowAt: indexPath)
```

Comment:
Sections with a integer backed enum would make it more readable

Reply:
Done